samples/tab-channel-group-config-page-auth/csharp/Views/Home/AuthEnd.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title>Simple Authentication Sample Login</title>
```

Comment:
Could you please add silent auth implementation as well? Provide options to choose from similar to Connector sample.

Reply:
Resolved